test/test_command_monitoring_legacy.py

```diff
@@ -47,7 +48,7 @@ class TestAllScenarios(unittest.TestCase):
     @client_context.require_connection
     def setUpClass(cls):
         cls.listener = EventListener()
-        cls.client = single_client(event_listeners=[cls.listener])
+        cls.client = MongoClient(event_listeners=[cls.listener])
```

Comment:
This needs to be `rs_or_single_client`, not MongoClient.

Reply:
Done.